docs/components/files/test-file-tiles.html

```diff
@@ -131,6 +131,18 @@ <h3>Downloadable with Custom Name</h3>
         <small>href defined</small>
       </div>
 
+      <div class="hxCol">
+        <h3>Downloadable Readonly</h3>
+        <hx-file-tile
+          details="50B"
+          href="files/easteregg.txt"
+          icon="mime-text"
+          name="downloadable-easter-egg.txt"
+          readonly
+        ></hx-file-tile>
+        <small>readonly download</small>
+      </div>
+
```

Comment:
Need additional tests:

- [ ] visually truncated + readonly (make sure name can extend all the way to the right edge)
- [ ] loading + readonly
- [ ] error + visually truncated + readonly

Reply:
@CITguy, I'm also adding an additional test to match the others:

- [ ] loading + visually truncated + readonly